presto-yarn-package/src/main/slider/package/scripts/configure.py

```diff
@@ -38,8 +38,9 @@ def set_configuration(component=None):
 
     _template_config("{params.conf_dir}/config.properties", params, component)
     _template_config("{params.conf_dir}/node.properties", params)
-    _template_config("{params.conf_dir}/queues.json", params)
```

Comment:
Please remove all the related files to query queues. Like https://github.com/prestodb/presto-yarn/blob/master/presto-yarn-package/src/main/slider/package/templates/queues.json.j2

Reply:
removed